tinylog-api/src/test/java/org/tinylog/provider/BundleLoggingProviderTest.java

```diff
@@ -124,8 +124,8 @@ public void isEnabled() {
 	}
 
 	/**
-	 * Verifies that {@code isEnabled} method evaluates the severity level from underlying logging providers and
-	 * returns {@code true} if given severity level is enabled at least for one of the underlying logging providers.
+	 * Verifies that {@code isEnabled()} method evaluates the severity level from underlying logging providers and

```

Comment:
Please remove the word "method". It is redundant.

Reply:
You are absolutely right about the consistency. Unfortunately, the Javadoc of the other method is not written in good English.